spec/features/reply_from_councillor_spec.rb

```diff
@@ -14,9 +15,8 @@
     end
   end
 
-  background { comment.confirm! }
-
   scenario "it's entered by an admin and displayed on the application page" do
+    comment.confirm!
```

Comment:
That's awesome @davwood thanks for explaining. In future would be cool to chuck that kinda thing on the commit message :)